packages/hooks/src/useAvailableWallets.ts

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import useWalletList from './useWalletsList';
+
+const useAvailableWallets = () => {
+    const { client } = useStore();
+    const { accounts, loginid, is_crypto } = client;
+    const { data, ...rest } = useFetch('authorize', {
+        payload: { authorize: accounts[loginid || ''].token },
+        options: { enabled: Boolean(loginid) },
+    });
+
+    // @ts-expect-error Need to update @deriv/api-types to fix the TS error
+    const { data: account_type_data } = useFetch('get_account_types', {
+        payload: { company: data?.authorize?.landing_company_name },
+        options: { enabled: Boolean(data?.authorize?.landing_company_name) },
+    });
+
+    const { data: added_wallets } = useWalletList();
+
+    const sortedWallets = React.useMemo(() => {
+        if (!account_type_data) return null;
+        // @ts-expect-error Need to update @deriv/api-types to fix the TS error
+        const { crypto, doughflow } = account_type_data?.get_account_types?.wallet;
```

Comment:
```suggestion
        const { crypto, doughflow } = (account_type_data?.get_account_types?.wallet || {});
```
You can add a fallback to this to fix the sonarcloud issue i think. Cause if any of those data returns undefined, it will short circuit and return a type error.